tox.ini

```diff
@@ -25,3 +25,22 @@ commands = make test-units
 
 [pycodestyle]
 ignore = E501
+
+[testenv:build-snap]
+base =
+skip_install = true
+labels = test, test-spread
+allowlist_externals = snapcraft
+commands = snapcraft --use-lxd
+
+[testenv:test-spread]
+base =
+depends = build-snap
+skip_install = true
+labels = test, test-spread
+allowlist_externals = git, spread
+commands_pre =
+    python -c "import os, pathlib, shutil;shutil.copy(sorted(pathlib.Path().glob('rockcraft_*.snap'), key=os.path.getmtime, reverse=True)[0], 'tests/rockcraft_spread.snap')"
```

Comment:
Can we check whether only 1 `rockcraft_*.snap` exists? In case there are multiple, the sorting alone might work but might use an unexpected snap.
An even better idea might be to first check whether `tests/rockcraft_*.snap`  exists, and then skip the copying. That way if I have multiple rockcraft snaps (which is usually the case for me) I can copy the correct one over before calling this.